drivers/ethernet/eth_sam_gmac.c

```diff
@@ -340,12 +358,17 @@ static inline void eth_sam_gmac_init_qav(Gmac *gmac)
 
 #endif
 
+#ifdef __DCACHE_PRESENT
 /*
```

Comment:
Nit; can you move the cache helpers up so we only need one #ifdef __DCACHE_PRESENT / #else / #endif block?

Reply:
@nandojve Could you look into this since I took this commit from your PR?

Reply:
Like this 9518832c.

Reply:
Rebased https://github.com/zephyrproject-rtos/zephyr/commit/9518832caafcd0d1f512ca542f7482a01a5a7bd9 into this PR.

Fixed.